src/main/java/gcewing/architecture/common/block/BlockShape.java

```diff
@@ -183,6 +183,10 @@ protected ArrayList<ItemStack> getDropsFromTileEntity(IBlockAccess world, BlockP
         ArrayList<ItemStack> result = new ArrayList<>();
         if (te instanceof TileShape ste) {
             ItemStack stack = ste.shape.kind.newStack(ste.shape, ste.baseBlockState, 1);
+
+            if (state.getBlock().getUnlocalizedName().contains("shapeSE")) {
+                stack = ste.shape.kind.newStack(ste.shape, ste.baseBlockState, 1, true);
+            }
```

Comment:
This can be done much simpler:
```suggestion
            ItemStack stack = ste.shape.kind.newStack(ste.shape, ste.baseBlockState, 1, state.getValue(LIGHT) > 0);
```

Reply:
simply using the light value causes materials that have a natural light level(such as glowstone) to drop the Shader Emissive variant of the block from the standard version, it can still be reduced to
```suggestion
            ItemStack stack = ste.shape.kind.newStack(ste.shape, ste.baseBlockState, 1, state.getBlock().getUnlocalizedName().contains("shapeSE"));
```